tutorials/keras_json_usage_1.md

```diff
@@ -73,3 +73,5 @@ You should be getting around 76-78% accuracy.
 
 This code trains and evaluates the loaded model on the dataset.
 
+### Code template
+[train.py](../example/train.py)
```

Comment:
Elaborate on what the code template does and the dataset it uses. Most importantly, write a one line sample usage code. Something like: `python train.py MODEL_NAME`. Do mention that the user has to type in the model name. Also, you can rename `train.py` to something like `keras_sample_cifar10.py`